server/src/main/java/org/apache/seata/server/controller/ClusterController.java

```diff
@@ -123,13 +122,11 @@ public MetadataResponse cluster(String group) {
     }
 
     @PostMapping("/watch")
-    public void watch(HttpServletRequest request, @RequestParam Map<String, Object> groupTerms,
-        @RequestParam(defaultValue = "28000") int timeout) {
-        AsyncContext context = request.startAsync();
```

Comment:
逻辑上要判断,如果是request为空就是走的8091,如果request不为空就是7091,要保留原逻辑
Logically, it should be judged that if the request is empty, it will go 8091, if the request is not empty, it will go 7091, and the original logic should be retained.